coral-component-actionbar/src/tests/test.ActionBar.js

```diff
@@ -652,19 +652,20 @@ describe('ActionBar', function() {
       bar.primary.items.add({}).appendChild(new Button());
      
       // Wait for resize listener
-      window.setTimeout(() => {
+      window.setTimeout(function() {
```

Comment:
What's the reason for using `function` over the arrow function ?

Reply:
Arrow function is only used for these `setTimeout`s. With test being flaky, I was trying to be consistent.